spec/features/twilio_posts_new_message_with_no_user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'Twilio' do
+  after do
+    page.driver.header 'X-Twilio-Signature', nil
```

Comment:
It makes sure the header used for twilio authentication is zeroed-out after test; recommended by twilio help docs.